src/css/responsive.css

```
@@ -0,0 +1,195 @@
+    padding: 0 8px;
+  }
+
+  /* Footer adjustments */
+  .footer {
+    padding: 12px 16px;
+  }
+
+  .footer p {
+    font-size: 0.7rem;
+    line-height: 1.5;
+    word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)